MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -208,6 +208,10 @@ public StreamBuilder(ITranscoderSupport transcoderSupport, ILogger logger)
 
                 var longBitrate = Math.Min(transcodingBitrate, playlistItem.AudioBitrate ?? transcodingBitrate);
                 playlistItem.AudioBitrate = longBitrate > int.MaxValue ? int.MaxValue : Convert.ToInt32(longBitrate);
+                if (playlistItem.AudioCodecs.Count == 0 && !string.IsNullOrWhiteSpace(transcodingProfile.AudioCodec))
+                {
+                    playlistItem.AudioCodecs = [transcodingProfile.AudioCodec];
```

Comment:
`transcodingProfile.AudioCodec` can be a list, so we should use `ContainerHelper.Split`.

Reply:
Not really valid for pure audio profiles though, specifying multiple codecs is also undefined behavior because the whole audio transcoding pipeline wants explicitly one codec per profile.

Reply:
If the client specifies codecs as a comma-separated list, the CanEncodeToAudioCodec test will immediately fail. We don’t support such profiles at this time, as handling them would require a significant rewrite of the audio transcoding pipeline, and we haven't yet identified a valid use case for this.